blocks/page_list/templates/itinerary/Header.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+export default class Header extends React.Component {
```

Comment:
Bad naming - avoid collisions with default html element names. <Header> is an existing HTML5 tag.